src/SortableItem/dragSpec.js

```diff
@@ -34,3 +34,7 @@ export function isDragging({ rowId }, monitor) {
 
    return rowId === draggingRowId;
 }
+
+export function canDrag(props, monitor) {
```

Comment:
1) where is this function used?
2) assuming that dndDisabled is already a boolean, we probably don't need the ternary here. If we expect it to be undefined `return !!props.dndDisabled` is more concise. Can also supply a default value in the params

Reply:
ah haha it is a bool, don't know why I did that

The canDrag function is gets imported with the rest of the DragSpec options by the SortableList component. It is a weird quirk of the React DnD library that it uses these optional functions to create a draggable higher order component.

See `src/SortableList/index.js` ln. 139